### ISSUE TITLE
Feature request: add ability to listen to a note, and set that as reference frequency

### ISSUE THREAD

**Issue by maxhambraeus** [thetwom/Tuner#79]
I could imagine this being useful when trying to tune your instrument the same as another. I guess you could do that by ear, but it would be nice to have the feature in the app.

**Comment by maxhambraeus** [comment]
Yeah, that's what I've been doing, but as you said it's pretty imprecise. Maybe showing decimals in scientific mode should also be an option?

**Comment by thetwom** [comment]
Hm, maybe we should show one dezicimal place? But still, I agree, that having the option to directly apply it to the reference note would be nice.

**Comment by maxhambraeus** [comment]
I think showing one decimal place is a good start

**Comment by thetwom** [comment]
I released [v6.7.0](https://github.com/thetwom/Tuner/releases/tag/v6.7.0) with the extra decimal place as a start :-). Should appear on fdroid and google play within the next few days.

**Comment by thetwom** [comment]
It has been quite some time, but finally I took the time to implement this request (see 93498488470fc8fcc2555751f8e1504c7baa760a). I will release this in the next few days ...

**Comment by thetwom** [comment]
This is now in release [v8.2.0](https://codeberg.org/thetwom/Tuner/releases/tag/v8.2.0). Note development shifted now from github.com to codeberg.org. Should become available on google play and fdroid in the next days ...

**Comment by maxhambraeus** [comment]
Just tested it from release apk, seems to work well! Moving to codeberg is also a fantastic idea in my opinion.
